js/apps/admin-ui/src/client-scopes/details/ScopeForm.tsx

```diff
@@ -55,6 +57,15 @@ export const ScopeForm = ({ clientScope, save }: ScopeFormProps) => {
     defaultValue: "false",
   });
 
+  const selectedProtocol = useWatch({
+    control,
+    name: "protocol",
+    defaultValue: clientScope?.protocol ?? providers[0],
```

Comment:
Is it needed to provide a default value here? Should the form itself not already have the correct value? Try doing some testing without this line, it might just work fine.
```suggestion
```